utils/benchmark/createImaAppTests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Why don't we run benchmark test inside of this file also?

Reply:
benchmark test moved into `createImaAppTests.sh`